src/Core/Content/Product/Subscriber/ProductSubscriber.php

```diff
@@ -53,7 +53,7 @@ private function sortProperties(SalesChannelProductEntity $product): PropertyGro
             if (!$origin) {
                 continue;
             }
-            $group = clone $origin;
+            $group = $sorted[$origin->getId()] ?? $sorted[$origin->getId()] = clone $origin;
```

Comment:
Please do not use the shorthand form here since it doesn't improve readability in this case.